src/components/shopping.js

```diff
@@ -16,25 +16,35 @@ function Shopping(props) {
       });
   }
 
-  return (
-    <FirestoreCollection
-      path={ITEMS}
-      filter={[USER_TOKEN, '==', props.userToken]}
-      render={({ data }) => {
-        return (
-          <div>
-            <h1>Shopping List</h1>
-            <ul>
-              {data.map(item => (
-                <li key={item.id} onClick={() => handleClick(item.id)}>
-                  {item.name} / {item.next_purchase}
-                </li>
-              ))}
-            </ul>
-          </div>
-        );
-      }}
-    />
-  );
+  if (props.list.length > 0) {
```

Comment:
Not a blocker, just a React-y comment to think about later. You'll rarely see `if/else` returning JSX. More often you'll see the idiomatic React way to conditionally render is something like:

```jsx
return (
  {props.list.length > 0 
    ? <ul>{props.list.map(...renderListItem)}</ul>
	: <div><p>Your list is empty!</p></div>
  }
)
```

Reply:
this was resolved when merging